### DISCUSSION TITLE
Feature request - Pop out driver/engineer messages

### DISCUSSION THREAD

**Discussion by beerman-uk** [f1multiviewer/issue-tracker#296] [Feature requests] [answered]
As mentioned in discord I use the free pixels below the 4 driver on boards for the RCM and the driver messages.  As the driver messages are on the same screen as the live timing, when the live timing refreshes it displays the top of the page meaning I would have to scroll down again to see them.  Could you please add a pop out for driver/engineer messages like we have for RCM?

**Comment by LapsTimeOFF** [comment, marked as answer]
Resolved
